ape_trezor/client.py

```diff
@@ -0,0 +1,108 @@
+from typing import Optional, Tuple
+
+from ape.api.accounts import TransactionAPI
+from ape.types import TransactionSignature
+from eth_typing.evm import ChecksumAddress
+from trezorlib import ethereum
+from trezorlib.client import get_default_client  # type: ignore
+from trezorlib.exceptions import PinException
+from trezorlib.tools import parse_path as parse_hdpath  # type: ignore
+from trezorlib.transport import TransportException
+
+from ape_trezor.exceptions import TrezorAccountException, TrezorUsbError
+from ape_trezor.hdpath import HDAccountPath, HDBasePath
+
+
+class TrezorClient:
+    """
+    This class is a client for the Trezor device.
+    """
+
+    def __init__(self, hd_root_path: HDBasePath):
+        try:
+            self._client = get_default_client()
+        except TransportException as exc:
+            message = (
+                "Unable to open Trezor device path."
```

Comment:
```suggestion
                "Unable to open Trezor device path. "
```

^ this has to end in a whitespace character